safeController:     "SafeControllerService/RateController"
defaultController:  "DefaultControllerService/DefaultController"
demoController:     "DemoControllerService/DemoController"
studentController:  "StudentControllerService/StudentController"
mpcController:      "MpcControllerService/MpcController"
remoteController:   "RemoteControllerService/RemoteController"
tuningController:   "TuningControllerService/TuningController"
pickerController:   "PickerControllerService/PickerController"
templateController: "TemplateControllerService/TemplateController"

# Flag for whether to use angle for switching to the Default Controller
isEnabled_strictSafety: true
maxTiltAngle_for_strictSafety_degrees: 70

battery_threshold_while_flying: 2.60       # in V
battery_threshold_while_motors_off: 3.30   # in V
battery_polling_period: 200                # in ms